homeassistant/components/stream/worker.py

```diff
@@ -141,6 +141,16 @@ def __init__(
         self._stream_settings = stream_settings
         self._stream_state = stream_state
         self._start_time = datetime.datetime.utcnow()
+        self._frag_duration_ts = (
+            self._stream_settings.part_target_duration * 0.85 / video_stream.time_base
```

Comment:
Can you describe what the 0.85 is or give it a constant or both?

After writing this, i see now it is described below with `Create a fragment every TARGET_PART_DURATION...` so maybe just pulling that whole description next to a constant can help connect them event though the variables are split between here and in the settings below.